CMakeTests.cmake

```diff
@@ -23,6 +23,9 @@
   math (EXPR CTEST_LONG_TIMEOUT "${DART_TESTING_TIMEOUT} * 2")
   math (EXPR CTEST_VERY_LONG_TIMEOUT "${DART_TESTING_TIMEOUT} * 3")
 
+  option (HDF5_DISABLE_TESTS_REGEX "Pattern to set tests to DISABLED" "")
```

Comment:
Maybe "exclude" would be more precise than "disable"

Reply:
#3863 created

Reply:
Not really, the tests are being skipped by disabling the execution of the tests.

Reply:
I think making things clear for the user should be a priority. So far, we have used "ENABLE" and "DISABLE" as boolean values.

at least: "Pattern used to mark specific tests as DISABLED"

Reply:
Changed option text

Reply:
@brtnfld, is this good now?